lib/Model/IMAPMessage.php

```diff
@@ -644,7 +645,10 @@ public function toDbMessage(int $mailboxId): \OCA\Mail\Db\Message {
 		$msg->setFlagFlagged(in_array(Horde_Imap_Client::FLAG_FLAGGED, $flags, true));
 		$msg->setFlagSeen(in_array(Horde_Imap_Client::FLAG_SEEN, $flags, true));
 		$msg->setFlagForwarded(in_array(Horde_Imap_Client::FLAG_FORWARDED, $flags, true));
-		$msg->setFlagJunk(in_array(Horde_Imap_Client::FLAG_JUNK, $flags, true));
+		$msg->setFlagJunk(
+			in_array(Horde_Imap_Client::FLAG_JUNK, $flags, true) ||
+			in_array('junk', $flags, true)
```

Comment:
Where's this flag from?
https://mailarchive.ietf.org/arch/msg/morg/m6dRIwLStfK__7aIHNJtOTPqxfU/

Reply:
Just saw your comment

Reply:
It's the deprecated keyword, but I found that it's still being used on my personal account. I can't say if it's written by the mail server or another client, but I think it makes sense to try to stay compatible with that as well.